plugins/couchbase/couchbaseembed.cpp

```diff
@@ -1196,3 +1531,30 @@ namespace couchbaseembed
         return true; // TO-DO
     }
 } // namespace
+
+MODULE_INIT(INIT_PRIORITY_STANDARD)
```

Comment:
Not sure what you mean, here.  MODULE_INIT and MODULE_EXIT are both top-level constructs outside the namespace (which is at that same level), and the code within the constructs seems to be indented correctly.